.github/workflows/nodejs.yml

```diff
@@ -57,6 +57,9 @@ jobs:
         node-version: [10.x, 12.x, 14.x]
         webpack-version: [4, latest]
         dev-server-version: [latest, next]
```

Comment:
why not change this instead?

Reply:
`dev-server-version: [latest]`

Reply:
will do after stable v4 release, can't do it till then